configs/webpack.config.base.js

```diff
@@ -64,6 +64,7 @@ export default {
   plugins: [
     new webpack.EnvironmentPlugin({
       NODE_ENV: 'production',
+      SHA: '',
```

Comment:
This missing property value wouldn't be best caught downstream instead of setting to an empty string?

Reply:
This is just a default, empty is okay—good question though